### PR TITLE
Julia 0.7

### PR DESCRIPTION
**Pull request by MikeInnes**
* [ ] Update CUDAnative.default_device for GPUArrays
* [ ] fix broadcasting
* [ ] fix BLAS operations

### CODE REVIEW

File: src/memory.jl [hunk 1]
```diff
@@ -8,7 +10,7 @@
 # - when requested memory, check the pool for unused memory, or allocate dynamically
 # - conversely, when released memory, put it in the appropriate pool for future use
 #
-# to avoid memory hogging and/or trashing the Julia GC:
+# to aNothing memory hogging and/or trashing the Julia GC:
```
Comment (CarloLucibello) [inline src/memory.jl]: typo

Comment (MikeInnes) [reply src/memory.jl]: thanks :+1:

**Comment by maleadt** [comment]
I added CI.

**Comment by maleadt** [comment]
Contained in https://github.com/JuliaGPU/CuArrays.jl/pull/105

**Pull request closed without merging**